ethminer/MinerAux.h

```diff
@@ -464,7 +464,11 @@ class MinerCLI
 			PhoneHome rpc(client);
 			try
 			{
+#ifdef __INTEL_COMPILER
+				unsigned ranking = rpc.report_benchmark(platformInfo, (int)innerMean.convert_to<int>());
```

Comment:
Would this work without the C-style cast?
So in all of these cases (and in the other PR too), we would ditch the #ifdef, and just switch the C-cast to the convert_to<> unconditionally, so this particular one, for example would go from ...

unsigned ranking = rpc.report_benchmark(platformInfo, (int)innerMean);

to ...

unsigned ranking = rpc.report_benchmark(platformInfo, innerMean.convert_to<int>());